index/scorch/segment/zap/new.go

```diff
@@ -714,6 +731,9 @@ func (s *interim) writeDicts() (fdvIndexOffset uint64, dictOffsets []uint64, err
 
 			tfEncoder.Reset()
 			locEncoder.Reset()
+			lastDocNum = 0
+			lastFreq = 0
+			lastNorm = 0
```

Comment:
actually these are either in the wrong place, or not necessary in this verison, as these variables go out of scope here...